scripts/android/autopower/README.md

```diff
@@ -1,26 +1,47 @@
 The "autopower.py" script measures the current used by an Android device
 while running SynthMark.
 
-Required
-
+### Requirements
 * phone with battery replacement cables
```

Comment:
We also need to install those two python packages, gflags and serial

Reply:
Those are dependencies of monsoon.py, but I will add it as a note.